vspace/src/main/java/edu/asu/diging/vspace/web/general/search/PublicSearchController.java

```diff
@@ -0,0 +1,118 @@
+package edu.asu.diging.vspace.web.general.search;
```

Comment:
there is a `publicpages` package in which all these controllers should probably live, or is there a reason for this new package?

Reply:
moved controllers to publicpages package